v3/zykrah/fuyu_hs.json

```diff
@@ -0,0 +1,347 @@
+        {
+          "d": true
+        },
+        "3,15\n\n\n4,1",
+        {
+          "d": true
+        },
+        "3,16\n\n\n4,1"
+      ],
+      [
+        {
+          "x": 2.5,
+          "c": "#aaaaaa",
+          "w": 2.25
+        },
+        "4,0",
+        {
+          "c": "#cccccc"
+        },
+        "4,2",
+        "4,3",
+        "4,4",
+        "4,5",
+        "4,6",
+        "4,7",
+        "4,8",
+        "4,9",
+        "4,10",
+        "4,11",
+        {
+          "c": "#aaaaaa",
+          "w": 2.75
+        },
+        "4,12\n\n\n2,0",
+        {
+          "x": 0.25,
+          "c": "#cccccc"
+        },
+        "4,14\n\n\n4,0",
+        "4,15",
+        "4,16\n\n\n4,0",
+        {
+          "x": 0.25,
+          "c": "#aaaaaa",
+          "w": 1.75
+        },
+        "4,12\n\n\n2,1",
+        "4,13\n\n\n2,1",
+        {
+          "x": 0.25,
+          "c": "#cccccc",
+          "d": true
+        },
+        "4,14\n\n\n4,1",
+        {
+          "x": 1,
+          "d": true
+        },
+        "4,16\n\n\n4,1"
+      ],
+      [
+        {
+          "x": 2.5,
+          "c": "#aaaaaa",
+          "w": 1.25
+        },
+        "5,0\n\n\n1,0",
+        {
+          "w": 1.25
+        },
+        "5,1\n\n\n1,0",
+        {
+          "w": 1.25
+        },
+        "5,2\n\n\n1,0",
+        {
+          "c": "#cccccc",
+          "w": 6.25
+        },
+        "5,6\n\n\n1,0",
+        {
+          "c": "#aaaaaa",
+          "w": 1.25
+        },
+        "5,10\n\n\n1,0",
+        {
+          "w": 1.25
+        },
+        "5,11\n\n\n1,0",
+        {
+          "w": 1.25
+        },
+        "5,12\n\n\n1,0",
+        {
+          "w": 1.25
+        },
+        "5,13\n\n\n1,0",
+        {
+          "x": 0.25,
+          "c": "#cccccc"
+        },
+        "5,14",
+        "5,15",
+        "5,16"
+      ],
+      [
+        {
+          "y": 0.25,
+          "x": 2.5,
+          "c": "#aaaaaa",
+          "w": 1.5
+        },
+        "5,0\n\n\n1,1",
+        "5,1\n\n\n1,1",
+        {
+          "w": 1.5
+        },
+        "5,2\n\n\n1,1",
+        {
+          "c": "#cccccc",
+          "w": 7
+        },
+        "5,6\n\n\n1,1",
+        {
+          "c": "#aaaaaa",
+          "w": 1.5
+        },
+        "5,11\n\n\n1,1",
+        "5,12\n\n\n1,1",
+        {
+          "w": 1.5
+        },
+        "5,13\n\n\n1,1"
+      ],
+      [
+        {
+          "x": 2.5,
+          "w": 1.5
+        },
+        "5,0\n\n\n1,2",
+        {
+          "d": true
+        },
+        "5,1\n\n\n1,2",
+        {
+          "w": 1.5
+        },
+        "5,2\n\n\n1,2",
+        {
+          "c": "#cccccc",
+          "w": 7
+        },
+        "5,6\n\n\n1,2",
+        {
+          "c": "#aaaaaa",
+          "w": 1.5
+        },
+        "5,11\n\n\n1,2",
+        {
+          "d": true
+        },
+        "5,12\n\n\n1,2",
+        {
+          "w": 1.5
+        },
+        "5,13\n\n\n1,2"
+      ]
+    ]
+  }
+}
```

Comment:
```suggestion
{
  "name": "Fuyu Hotswap",
  "vendorId": "0x7A79",
  "productId": "0x4648",
  "matrix": {
    "rows": 6,
    "cols": 17
  },
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_rgb_matrix"
  ],
  "layouts": {
    "labels": [
      "Split Backspace",
      "Split Right Shift",
      "Numpad Blocker",
      [
        "Bottom Row",
        "6.25U",
        "7U",
        "WKL"
      ]
    ],
    "keymap": [
      [
        {
          "c": "#777777"
        },
        "0,0",
        {
          "x": 1,
          "c": "#cccccc"
        },
        "0,2",
        "0,3",
        "0,4",
        "0,5",
        {
          "x": 0.5
        },
        "0,6",
        "0,7",
        "0,8",
        "0,9",
        {
          "x": 0.5
        },
        "0,10",
        "0,11",
        "0,12",
        "0,13",
        {
          "x": 0.25
        },
        "0,14",
        "0,15",
        "0,16"
      ],
      [
        {
          "y": 0.25
        },
        "1,0",
        "1,1",
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        "1,8",
        "1,9",
        "1,10",
        "1,11",
        "1,12",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "1,13\n\n\n0,0",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "1,14",
        "1,15",
        "1,16",
        {
          "x": 1
        },
        "1,13\n\n\n0,1",
        {
          "c": "#aaaaaa"
        },
        "3,13\n\n\n0,1"
      ],
      [
        {
          "w": 1.5
        },
        "2,0",
        {
          "c": "#cccccc"
        },
        "2,1",
        "2,2",
        "2,3",
        "2,4",
        "2,5",
        "2,6",
        "2,7",
        "2,8",
        "2,9",
        "2,10",
        "2,11",
        "2,12",
        {
          "w": 1.5
        },
        "2,13",
        {
          "x": 0.25
        },
        "2,14",
        "2,15",
        "2,16"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "3,0",
        {
          "c": "#cccccc"
        },
        "3,1",
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        "3,7",
        "3,8",
        "3,9",
        "3,10",
        "3,11",
        {
          "c": "#777777",
          "w": 2.25
        },
        "3,12",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "3,14\n\n\n2,0",
        "3,15\n\n\n2,0",
        "3,16\n\n\n2,0",
        {
          "x": 3.25,
          "d": true
        },
        "3,14\n\n\n2,1",
        {
          "d": true
        },
        "3,15\n\n\n2,1",
        {
          "d": true
        },
        "3,16\n\n\n2,1"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 2.25
        },
        "4,0",
        {
          "c": "#cccccc"
        },
        "4,2",
        "4,3",
        "4,4",
        "4,5",
        "4,6",
        "4,7",
        "4,8",
        "4,9",
        "4,10",
        "4,11",
        {
          "c": "#aaaaaa",
          "w": 2.75
        },
        "4,12\n\n\n1,0",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "4,14\n\n\n2,0",
        "4,15",
        "4,16\n\n\n2,0",
        {
          "x": 0.25,
          "c": "#aaaaaa",
          "w": 1.75
        },
        "4,12\n\n\n1,1",
        "4,13\n\n\n1,1",
        {
          "x": 0.25,
          "c": "#cccccc",
          "d": true
        },
        "4,14\n\n\n2,1",
        {
          "x": 1,
          "d": true
        },
        "4,16\n\n\n2,1"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "5,0\n\n\n3,0",
        {
          "w": 1.25
        },
        "5,1\n\n\n3,0",
        {
          "w": 1.25
        },
        "5,2\n\n\n3,0",
        {
          "c": "#cccccc",
          "w": 6.25
        },
        "5,6\n\n\n3,0",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "5,10\n\n\n3,0",
        {
          "w": 1.25
        },
        "5,11\n\n\n3,0",
        {
          "w": 1.25
        },
        "5,12\n\n\n3,0",
        {
          "w": 1.25
        },
        "5,13\n\n\n3,0",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "5,14",
        "5,15",
        "5,16"
      ],
      [
        {
          "y": 0.25,
          "c": "#aaaaaa",
          "w": 1.5
        },
        "5,0\n\n\n3,1",
        "5,1\n\n\n3,1",
        {
          "w": 1.5
        },
        "5,2\n\n\n3,1",
        {
          "c": "#cccccc",
          "w": 7
        },
        "5,6\n\n\n3,1",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "5,11\n\n\n3,1",
        "5,12\n\n\n3,1",
        {
          "w": 1.5
        },
        "5,13\n\n\n3,1"
      ],
      [
        {
          "y": 0.25,
          "w": 1.5
        },
        "5,0\n\n\n3,2",
        {
          "d": true
        },
        "5,1\n\n\n3,2",
        {
          "w": 1.5
        },
        "5,2\n\n\n3,2",
        {
          "c": "#cccccc",
          "w": 7
        },
        "5,6\n\n\n3,2",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "5,11\n\n\n3,2",
        {
          "d": true
        },
        "5,12\n\n\n3,2",
        {
          "w": 1.5
        },
        "5,13\n\n\n3,2"
      ]
    ]
  }
}

```

Reply:
Removed stepped caps as it doesn't provide any actual alternative layout being in the same matrix position.
General cleanup